### PR TITLE
MOB-1986 - Activate MPC wallet during onboarding

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Modules/MPC/EnterCode/MPCOnboardingEnterCodeViewController.swift [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 
-final class MPCEnterPassphraseViewController: BaseViewController, ViewWithDashesProgress {
+final class MPCOnboardingEnterCodeViewController: BaseViewController, ViewWithDashesProgress {
     
     override var analyticsName: Analytics.ViewName { .onboardingMPCEnterCode }
     override var preferredStatusBarStyle: UIStatusBarStyle { .default }
     
     weak var onboardingFlowManager: OnboardingFlowManager?
-    var progress: Double? { 1 / 2 }
+    var progress: Double? { 3 / 4 }
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Modules/MPC/EnterCode/MPCOnboardingEnterCodeViewController.swift:16]: 0.75 ?)

File: unstoppable-ios-app/domains-manager-ios/Modules/Onboarding/OnboardingAddWallet/OnboardingAddWalletType.swift [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            return "MPC Wallet"
+        case .selfCustody:
+            return String.Constants.selfCustody.localized()
+        }
+    }
+    
+    var subtitle: String? {
+        switch self {
+        case .mpcWallet:
+            return "Full cryptocurrency wallet for Web3 with enhanced backup."
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Modules/Onboarding/OnboardingAddWallet/OnboardingAddWalletType.swift:34]: Extract all string constants in this file?

File: unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -189,7 +191,7 @@
 // Creating wallet
 "CREATING_WALLET" = "Creating wallet";
 "CREATE_NEW_VAULT_TITLE" = "Create new wallet";
-"CREATE_NEW_VAULT_SUBTITLE" = "A wallet is a cryptocurrency address for your Web3 identity.\nUse your wallet to track your crypto assets including NFTs, manage domains and connect to Web3 apps.";
+"CREATE_NEW_VAULT_SUBTITLE" = "Wallet used to store your domains, crypto and collectibles.";
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/SupportingFiles/Localization/en.lproj/Localizable.strings:194]: wallet IS used to....

File: unstoppable-ios-app/domains-manager-ios/Modules/Onboarding/OnboardingAddWallet/OnboardingAddWalletType.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Modules/Onboarding/OnboardingAddWallet/OnboardingAddWalletType.swift:34]: This module is not finalised yet. Will be extracted when scope and copies finalised

**Pull request merged**